src/game/game_main.cpp

```diff
@@ -76,7 +76,7 @@ void setup() {
         V2(0.0, 0.0),
     };
     Physics::add_shape(LEN(points), points);
-    Renderer::set_window_size(500, 500);
+    Renderer::set_window_size(1000, 1000);
```

Comment:
You do know that not everyone has a 1000x1000 monitor right?

Reply:
Sorry, at 500 I have trouble reading the text. Maybe it could be a command-line flag so we don't edit it back and forth all the time?

Reply:
Sounds like we need HighDPI settings